src/disable.coffee

```diff
@@ -0,0 +1,83 @@
+_ = require 'underscore-plus'
+path = require 'path'
+CSON = require 'season'
+yargs = require 'yargs'
+
+config = require './apm'
+Command = require './command'
+List = require './list'
+
+module.exports =
+class Disable extends Command
+  @commandNames: ['disable']
+
+  parseOptions: (argv) ->
+    options = yargs(argv).wrap(100)
+    options.usage """
+
+      Usage: apm disable [<package_name>]...
+
+      Disables the named package(s).
+    """
+    options.alias('h', 'help').describe('help', 'Print this usage message')
+
+  getInstalledPackages: (callback) ->
+    options =
+      argv:
+        theme: false
+        bare: true
+
+    lister = new List()
+    lister.listBundledPackages options, (error, core_packages) ->
+      lister.listDevPackages options, (error, dev_packages) ->
+        lister.listUserPackages options, (error, user_packages) ->
+          callback(null, core_packages.concat(dev_packages, user_packages))
+
+  run: (options) ->
+    {callback} = options
+    options = @parseOptions(options.commandArgs)
+
+    packageNames = @packageNamesFromArgv(options.argv)
+
+    configFilePath = CSON.resolve(path.join(config.getAtomDirectory(), 'config'))
+    unless configFilePath
+      callback("Could not find config.cson. Run Atom first?")
```

Comment:
It might be a good idea to add a spec for this case (when there is no `config.cson`)